python/surf/protocols/pgp/_Pgp3AxiL.py

```diff
@@ -14,6 +14,7 @@
 #-----------------------------------------------------------------------------
 
 import pyrogue as pr
+import surf.xilinx as xil
```

Comment:
I know it's subjective, but I'm not a fan of the `import as` style as applied here. Especially if there is only 1 use of the imported package in the code below. Leaving the packages fully named makes it easier to see which library/package things are coming from as you browse the code, without having to refer back to the `import` as the top to resolve the abbreviation.

It can sometimes be useful to avoid verbosity, but that is not the case here.

Reply:
@bengineerd: Feel free to do this change on this PR